<launch>
    <group ns="stereo">
        <node name="videre_stereo_cam" pkg="videre_stereo_cam" type="videre_stereo_node" respawn="true" output="screen">
            <param name="fps" type="double" value="30"/>
            <param name="frame_id" value="stereo_optical_frame"/>
            <param name="videre_mode" type="str" value="none"/>
            <param name="convert_to_color" type="bool" value="False"/>
            
            <!--param name="exposure" type="int" value="450"/-->
            <param name="exposure_auto" type="bool" value="True"/>
            <!--param name="brightness" type="int" value="50"/-->
            <param name="brightness_auto" type="bool" value="True"/>
            <!--param name="gain" type="int" value="10"/-->
            <param name="gain_auto" type="bool" value="True"/>
        </node>
        <group ns="left">
        	<node name="image_proc" pkg="image_proc" type="image_proc"  output="screen">
			<param name="disparity_range" type="double" value="112.0"/>
		</node>
	</group>
    </group>
</launch>


